src/tasks/helpers/replacers/UriParameters.ts

```diff
@@ -30,13 +30,40 @@ const replaceable: ASTReplaceable = {
 		if (oInsertion.type === Syntax.CallExpression) {
 			const oldArgs = oInsertion.arguments;
 			const newArgs = [];
-			if (oldArgs.length === 0) {
-				if (config.functionParameter) {
-					const oAst = recast.parse(config.functionParameter).program
-						.body["0"].expression;
-					newArgs.push(oAst);
-				}
+
+			const windowLocationSearch = builders.memberExpression(
+				builders.memberExpression(
+					builders.identifier("window"),
+					builders.identifier("location")
+				),
+				builders.identifier("search")
+			);
+			const fromQuery = builders.memberExpression(
+				builders.identifier(name),
+				builders.identifier("fromQuery")
+			);
+			const fromURL = builders.memberExpression(
+				builders.identifier(name),
+				builders.identifier("fromURL")
+			);
+
+			let oNodeModule;
+
+			// jQuery.sap.getUriParameters() --> UriParameters.fromQuery(window.location.search)
+			// jQuery.sap.getUriParameters(window.location.href) --> UriParameters.fromQuery(window.location.search)
```

Comment:
The same applies to just `location.href` and `location.search`.

Do we have the capability to check that the first name is a global name and not overridden locally? Might be over-engineered to check this...

Reply:
that's a good point. I would include `location.href` and `location.search`.

Within the `replacer` only the current node is accessible. There is a `parentPath` chain though which could be walked up and each parent could be processed to find out if a global variable is overridden. But imho this should not be done by the `replacer` and if so, then I would expect a better concept.

There are also other combinations... but I would leave these out.
```
< window.self.location.href
> "https://github.com/SAP/ui5-migration/pull/69"

< window.window.location.href
> "https://github.com/SAP/ui5-migration/pull/69"
```